### ISSUE TITLE
Text on homepage should be limited to 80-120 chars per line for readability

### ISSUE THREAD

**Issue by RichardLitt** [octobox/octobox#1159]
I want to suggest that you keep the text limited to around 80~ characters per line, but I was unsure where to suggest this. Where is the sourcecode for the website? There was no `gh-pages` branch that I could see in this repository, so I am unsure. 

Feel free to close if I've misfiled this. :)

**Comment by andrew** [comment]
This is the correct repo, the homepage for Octobox.io is the same as any other instance of this repo

**Comment by RichardLitt** [comment]
Ah! You're right! I misread something, somewhere. Thanks. In that case, renaming this issue.

**Comment by BenJam** [comment]
I completely agree and have this in mind for a re-implementation of the homepage which I'll get to asap.

**Comment by BenJam** [comment]
@RichardLitt feel free to take a look at #1187 and let me know whether it meets your discerning expectations :)

**Comment by RichardLitt** [comment]
Looks much better to me. :) Thank you, @BenJam. Sorry for the delay!
